pkgs/applications/networking/instant-messengers/vk-messenger/default.nix

```diff
@@ -0,0 +1,53 @@
+{ stdenv, fetchurl, dpkg, autoPatchelfHook
+, xorg, gtk2, gnome2, nss, alsaLib
+, udev, libnotify }:
+
+stdenv.mkDerivation rec {
+  name = "vk-messenger-3.9.0";
+  src = {
+    name = "${name}.deb";
+    i686-linux = fetchurl {
+      url = "https://desktop.userapi.com/get_last?platform=linux32&branch=master&packet=deb";
```

Comment:
`get_last`? `master`?? The hash will change which will break the package.

Reply:
@dotlambda ok, I'll try to find packages archives

Reply:
@dotlambda 
As of urls
https://desktop.userapi.com/debian/pool/master/v/vk/
https://desktop.userapi.com/rpm/master/
https://desktop.userapi.com/linux64/master/

Only https://desktop.userapi.com/rpm/master/ keeps archives so I took that one. Unfortunately it doesn't ship arm version, so I dropped support for arm